Corona-Warn-App/src/main/res/layout/contact_diary_location_list_item.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/contact_diary_location_list_line"
```

Comment:
May rename the ids to "[...]_list_item" instead of "[...]_list_line to be aligned with the other files

Reply:
done